EasyPost.Integration/Basics.cs

```diff
@@ -86,4 +86,54 @@ public void TestUserCanLocallyConstructResponseObject()
         var verifications = new Verifications();
         var webhook = new Webhook();
     }
+
+    /// <summary>
+    ///     Test that an end-user can locally construct all available hooks.
+    ///     If this test can be compiled, then the hooks are publicly accessible.
+    /// </summary>
+    [Fact, Testing.Access, Testing.Compile]
+    public void TestUserCanCreateHooks()
+    {
+        // Can set up each hook event handler during construction
+        var hooks = new Hooks()
+        {
+            OnRequestResponseReceived = new EventHandler<OnRequestResponseReceivedEventArgs>((sender, args) =>
+            {
+                Console.WriteLine(args.Response);
+                Console.WriteLine(args.RequestTimestamp);
+                Console.WriteLine(args.ResponseTimestamp);
+                Console.WriteLine(args.Id);
```

Comment:
issue: these should not be writing out. We should be asserting like a unit test. Integration tests are meant to be automated as are unit tests but they test the integration of the parts. Having it print lines like this requires user intervention which I promise we will stop doing 2 weeks from now when we forget this does that.